src/main/kotlin/org/koil/image/ImageNormalizer.kt

```diff
@@ -0,0 +1,39 @@
+package org.koil.image
+
+import org.imgscalr.Scalr
+import org.springframework.stereotype.Component
+import java.io.File
+import java.io.FileInputStream
+import java.io.InputStream
+import javax.imageio.ImageIO
+
+/**
+ * Image type. Format name corresponds to the format names recognized by java's ImageIO class.
+ */
+enum class ImageType(val formatName: String) {
+    PNG("png")
+}
+
+data class ImageNormalizationParameters(
+    val width: Int = 150,
```

Comment:
Is there a reason to have chosen 150 as the default width? Is 150 just a good image size for a profile picture? (Trying to remember image sizes and I feel like 500 is more normal for a decent quality image)